plan/plans/show.go

```diff
@@ -528,12 +515,12 @@ func (s *ShowPlan) fetchShowStatus(ctx context.Context) error {
 
 		var value string
 		if !s.GlobalScope {
-			value, err = getSessionStatusVar(ctx, sessionVars, globalVars, v.Name)
+			value, err = getSessionStatusVar(ctx, sessionVars, status, v)
```

Comment:
There are some sessionVars values and statusVars values different situation . So I think it is needed.